### DISCUSSION TITLE
About distribution

### DISCUSSION THREAD

**Discussion by cristianvogel** [cables-gl/cables_docs#688] [Answers & Questions] [answered]
Hi.
I was thinking of wrapping a mostly FLOSS built product for commercial sale. Not expecting it to be big business, but I am not really completely clear if its ok to use a Cables front end  embedded in a commercial product? Could this be clarified?

**Comment by steam0r** [comment, marked as answer]
there is a section in the docs about this: https://cables.gl/docs/faq/licence_payment/userlicence/userlicence

tl;dr: yes, you can use work created in/with cabled in commercial projects
